src/constants.js

```diff
@@ -451,6 +451,10 @@ export const TASK_EDIT_MODE_EDIT: 'EDIT' = 'EDIT';
 /* ----------------- Task Validation ---------------- */
 export const TASK_MAX_GROUP_ASSIGNEES: 250 = 250;
 
+/* ------------------ Comment (and Annotation) Statuses ----------------- */
+export const COMMENT_STATUS_OPEN: 'open' = 'open';
+export const COMMENT_STATUS_RESOLVED: 'resolved' = 'resolved';
+
```

Comment:
Does it make sense to call this `FEED_ITEM_STATUS_[...]` so it aligns to with the `type FeedItemStatus`: https://github.com/box/box-ui-elements/blob/d0f669b4ad4323c5ea6f65151a1269f750d32f4b/src/common/types/feed.js#L27

Reply:
@tjuanitas, this is due to feedback from a prior PR. The "feed" is an experience rather than a resource, so referring to it in our API layer feels a bit backward. Using "comment" here is also more consistent with the other existing annotation/task/version constants in this file.

Reply:
it has been discussed here: https://github.com/box/box-ui-elements/pull/3004#discussion_r955121343